db/migrate/20190910163930_create_uphold_report.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUpholdReport < ActiveRecord::Migration[5.2]
+  def change
+    add_column :uphold_connections, :member_at, :datetime
+
+    create_table :uphold_reports, id: :uuid, default: -> { "uuid_generate_v4()"}, force: :cascade do |t|
+      t.belongs_to :publisher, index: true, type: :uuid
+      t.uuid :uphold_id
```

Comment:
Let's create an index here too to improve the above `find_by`

Reply:
https://github.com/brave-intl/publishers/pull/2219/commits/c35cb3d2f276a0c1b81ee111a38d9f4b56e8744e